packages/fx-core/src/plugins/resource/apiconnector/errors.ts

```diff
@@ -50,29 +59,29 @@ export class ErrorMessage {
 
   public static readonly NoValidCompoentExistError = {
     name: "NoValidCompoentExistError",
-    message: () =>
-      getLocalizedString(`error.apiConnector.${ErrorMessage.NoValidCompoentExistError.name}`),
+    message: (): [string, string] =>
+      ErrorMessage.getMessages(`error.apiConnector.${ErrorMessage.NoValidCompoentExistError.name}`),
   };
 
   public static readonly NoActivePluginsExistError = {
     name: "NoActivePluginsExistError",
-    message: () =>
-      getLocalizedString(`error.apiConnector.${ErrorMessage.NoActivePluginsExistError.name}`),
+    message: (): [string, string] =>
```

Comment:
can we make the return type an object? maybe something like:
```
{
  localizedErrorMessage: 'xxx',
  defaultErrorMessage: 'yyy'
}
```

This would make the code more clear and avoid possible errors

Reply:
updated